Update best-practices.rst (#35692)
mistercrunch/Airflow: docs/apache-airflow/best-practices.rst

          dag=dag,
      )

> What you can do to check it is add some print statements to the code you want to check and then run

Reviewer: ```suggestion
You can to check it by adding some print statements to the code, and running
```

Shorter and more readable imo.

Reviewer 2: I agree with @uranusjr
```suggestion
You can check it by adding some print statements to the code, and running
```
